grpc-gateway/pb/service.proto

```diff
@@ -151,6 +152,26 @@ service GrpcGateway {
     };
   }
 
+  // Cancels resource commands.
+  rpc CancelResourceCommands(CancelResourceCommandsRequest) returns (CancelResponse) {
+    option (google.api.http) = {
+      delete: "/api/v1/pending-commands/resources"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      tags: [ "Resource Directory" ]
+    };
+  }
+
+  // Cancels device metadata updates.
+  rpc CancelDeviceMetadataUpdates(CancelDeviceMetadataUpdatesRequest) returns (CancelResponse) {
```

Comment:
Did we decide corretly to have for DevceMetadata separate aggregate func? Yes, we can use protobuf only, but we cannot unify commands; we cannot have one cancel commands for all types of commands, I cannot cancel everything on one device *including* metadata of this device ... What do you think? It's a pitty.

Reply:
you can cancel all commands at one resource.

Reply:
we can change API to grpc-gw - which fetch all pending commands of the device and then cancel them by correlationID.
It means API for grpc will be:
```
{
  string device_id = 1;
  repeated string correlation_id_filter = 2;
}
```
But this is only for convenience - developer knows which cancel need to call.